[2025-09-20T15:06:25.619+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sap_elt_pipeline.run_dbt_models manual__2025-09-20T15:05:01.247658+00:00 [queued]>
[2025-09-20T15:06:25.647+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sap_elt_pipeline.run_dbt_models manual__2025-09-20T15:05:01.247658+00:00 [queued]>
[2025-09-20T15:06:25.648+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-20T15:06:25.694+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): run_dbt_models> on 2025-09-20 15:05:01.247658+00:00
[2025-09-20T15:06:25.713+0000] {standard_task_runner.py:60} INFO - Started process 2650 to run task
[2025-09-20T15:06:25.725+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sap_elt_pipeline', 'run_dbt_models', 'manual__2025-09-20T15:05:01.247658+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/sap_etl_pipeline.py', '--cfg-path', '/tmp/tmps5y2e1hd']
[2025-09-20T15:06:25.734+0000] {standard_task_runner.py:88} INFO - Job 62: Subtask run_dbt_models
[2025-09-20T15:06:25.877+0000] {task_command.py:423} INFO - Running <TaskInstance: sap_elt_pipeline.run_dbt_models manual__2025-09-20T15:05:01.247658+00:00 [running]> on host ca9121c9acbc
[2025-09-20T15:06:26.159+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sap_elt_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_models' AIRFLOW_CTX_EXECUTION_DATE='2025-09-20T15:05:01.247658+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-20T15:05:01.247658+00:00'
[2025-09-20T15:06:26.179+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-20T15:06:26.197+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt run --project-dir /opt/***/dbt --profiles-dir /opt/***/dbt']
[2025-09-20T15:06:26.234+0000] {subprocess.py:86} INFO - Output:
[2025-09-20T15:06:32.240+0000] {subprocess.py:93} INFO - [0m15:06:32  Running with dbt=1.9.0-b2
[2025-09-20T15:06:32.259+0000] {subprocess.py:93} INFO - [0m15:06:32  [[33mWARNING[0m]: Deprecated functionality
[2025-09-20T15:06:32.260+0000] {subprocess.py:93} INFO - 
[2025-09-20T15:06:32.263+0000] {subprocess.py:93} INFO - User config should be moved from the 'config' key in profiles.yml to the 'flags' key in dbt_project.yml.
[2025-09-20T15:06:36.219+0000] {subprocess.py:93} INFO - [0m15:06:36  Registered adapter: snowflake=1.8.0
[2025-09-20T15:06:39.231+0000] {subprocess.py:93} INFO - [0m15:06:39  Encountered an error:
[2025-09-20T15:06:39.232+0000] {subprocess.py:93} INFO - Compilation Error
[2025-09-20T15:06:39.233+0000] {subprocess.py:93} INFO -   Model 'model.dbt_sap_transformations.stg_customers' (models/staging/stg_customers.sql) depends on a source named 'RAW_STAGING.RAW_KNA1' which was not found
[2025-09-20T15:06:39.867+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-09-20T15:06:39.896+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-09-20T15:06:39.905+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=sap_elt_pipeline, task_id=run_dbt_models, execution_date=20250920T150501, start_date=20250920T150625, end_date=20250920T150639
[2025-09-20T15:06:39.951+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 62 for task run_dbt_models (Bash command failed. The command returned a non-zero exit code 2.; 2650)
[2025-09-20T15:06:39.985+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-09-20T15:06:40.036+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
